src/main/java/com/eclipsesource/v8/LibraryLoader.java

```diff
@@ -31,7 +31,7 @@ private static String computeLibraryShortName() {
         String base = "j2v8";
         String osSuffix = getOS();
         String archSuffix = getArchSuffix();
-        return base + "_" + osSuffix + "_" + archSuffix;
+        return base + "-" + osSuffix + "-" + archSuffix;
```

Comment:
this is weird, I never had this problem

Reply:
We experienced this both on Linux and OS X. Library is embedded in JAR with dashes instead of lower dashes.